syntax = "proto3";
package flowable;

option go_package = "go.easyops.local/contracts/protorepo-models/easyops/model/flowable";





/*
流程定义
*/
message FlowableProcessDefinition {
    /*
    流程定义ID
    */
    string id = 1 ;
    /*
    URL
    */
    string url = 2 ;
    /*
    KEY
    */
    string key = 3 ;
    /*
    版本
    */
    int32 version = 4 ;
    /*
    名称
    */
    string name = 5 ;
    /*
    描述
    */
    string description = 6 ;
    /*
    tenant ID
    */
    string tenantId = 7 ;
    /*
    部署ID
    */
    string deploymentId = 8 ;
    /*
    部署Url
    */
    string deploymentUrl = 9 ;
    /*
    流程图xml保存地址
    */
    string resource = 10 ;
    /*
    流程图保存地址
    */
    string diagramResource = 11 ;
    /*
    分类
    */
    string category = 12 ;
    /*
    图形注释是否定义
    */
    bool graphicalNotationDefinied = 13 ;
    /*
    是否挂起
    */
    bool suspended = 14 ;
    /*
    发起流程表单是否定义
    */
    bool startFormDefined = 15 ;
}

